### PR TITLE
fix links and heading on SVG home page

### PR DESCRIPTION
**Pull request by peterbe**
> What was wrong/why is this fix needed? (quick summary only)

2 flaws. And the link to the tutorial would also always be replaced at build-time. 

> Issue number (if there is an associated issue)

nope

> Anything else that could help us review it

I cleaned two of those Unicode non-breaking spaces in the text too.

### CODE REVIEW

File: files/en-us/web/svg/index.html [hunk 1]
```diff
@@ -17,15 +17,17 @@
...
+<p class="summary" style="border-top-width: 0; padding-top: 0;"><span class="seoSummary"><strong>Scalable Vector Graphics (SVG)</strong> are an <a href="/en-US/docs/Web/XML">XML</a>-based markup language for describing two-dimensional based <a href="https://en.wikipedia.org/wiki/Vector_graphics">vector graphics</a>.</span> As such, it's a text-based, open Web standard for describing images that can be rendered cleanly at any size and are designed specifically to work well with other web standards including <a href="/en-US/docs/Web/CSS">CSS</a>, <a href="/en-US/docs/Web/API/Document_Object_Model">DOM</a>, <a href="/en-US/docs/Web/JavaScript">JavaScript</a>, and <a href="/en-US/docs/Web/SVG/SVG_animation_with_SMIL">SMIL</a>. SVG is, essentially, to graphics what <a href="/en-US/docs/Web/HTML">HTML</a> is to text.</p>
```
Comment (Ryuno-Ki) [inline files/en-us/web/svg/index.html:22]: Do we still need this `class` and `style`?

**Review by Ryuno-Ki — APPROVED** [review]
r+wc

File: files/en-us/web/svg/index.html (hunk 1 above)
Comment (peterbe) [reply files/en-us/web/svg/index.html:22]: I can fix that, "while the iron is hot".

**Review by Ryuno-Ki — APPROVED** [review]
This should do it!

**Pull request merged**